crates/net/network-api/src/lib.rs

```diff
@@ -49,6 +49,9 @@ pub trait NetworkInfo: Send + Sync {
 
     /// Returns `true` if the network is undergoing sync.
     fn is_syncing(&self) -> bool;
+
+    /// Returns `true` if the network is undergoing an initial sync.
```

Comment:
This needs some additional context what this exactly represents wrt pipeline sync:
if this is a fresh node sync then this is expected to track the first complete pipeline run.